test/mapping.test.js

```diff
@@ -0,0 +1,108 @@
+process.env.NODE_ENV = 'test';
+require('should');
+require('./init');
+
+var async = require('async');
+
+var db;
+
+before(function() {
+  db = getDataSource();
+});
+
+describe('Mapping models', function() {
+  it('should honor the mssql settings for table/column', function(done) {
+
+    var schema =
+    {
+      'name': 'TestInventory',
+      'options': {
+        'idInjection': false,
+        'mssql': {
+          'schema': 'dbo', 'table': 'INVENTORYTEST'
+        }
+      },
+      'properties': {
+        'productId': {
+          'type': 'Number', 'id': true, generated: true,
+          'mssql': {
+            'columnName': 'PRODUCT_ID',
+            'dataType': 'uniqueidentifier',
+            'nullable': 'N'
+          }
+        },
+        'locationId': {
+          'type': 'String', 'required': true, 'length': 20,
+          'mssql': {
+            'columnName': 'LOCATION_ID', 'dataType': 'nvarchar',
+            'nullable': 'N'
```

Comment:
I find this style confusing. IMO you should either put all items on a single line, or else put each key/value pair on a new line.